HISTORY.md

```diff
@@ -5,7 +5,8 @@
 * Added BlobDB options to `ldb`
 
 ### Bug Fixes
-* * Fixed a data race on `versions_` between `DBImpl::ResumeImpl()` and threads waiting for recovery to complete (#9496)
+* Fixed a data race on `versions_` between `DBImpl::ResumeImpl()` and threads waiting for recovery to complete (#9496)
+* Fixed a bug due to db mutex release causing incorrect result returned for snapshot read.
```

Comment:
"db mutex release" - this isn't user API level or very concrete. Should we say an unlikely race condition and/or list the conditions? Maybe we say the symptom too: queries to snapshots created with this race condition may return old data, including resurfacing deleted data.

Reply:
Thanks @ajkr for the review. Updated HISTORY